sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -67,6 +71,8 @@ public ChatThreadAsyncClient getChatThreadClient(String chatThreadId) {
      * Creates a chat thread.
      *
      * @param options Options for creating a chat thread.
+     * @throws ChatErrorResponseException thrown if the request is rejected by server.
```

Comment:
nit; Using the word `thrown` reads weird after seeing `@throws` right before. This applies to all instances of this in the project's JavaDoc.
```suggestion
     * @throws ChatErrorResponseException if the request is rejected by server.
```